mantle/vendor/github.com/coreos/coreos-assembler/LICENSE

```diff
@@ -179,15 +178,15 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
+      boilerplate notice, with the fields enclosed by brackets "{}"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2015 xeipuuv
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
This looks incomplete?

Reply:
This is just copying the `LICENSE` file that already exists at the toplevel.  Also, this section of the file appears to be boilerplate - it's after the "   END OF TERMS AND CONDITIONS" section.

So I don't think this is a real issue.